### ISSUE TITLE
How was the class interpolation accomplished?

### ISSUE THREAD

**Issue by jcpeterson** [pfnet-research/sngan_projection#6]
There doesn't seem to be working code for this, and the network is conditional, so I'm not sure how it's possible.

**Comment by ajbrock** [comment]
Interpolate between embeddings for the batchnorm in the class-conditional generator.

**Comment by jcpeterson** [comment]
The "embedding" is the gamma and beta values for each node or something before that? Can this be done with a pretrained model in the repo?

**Comment by ajbrock** [comment]
Yes, gamma and beta are embeddings. If you have a pre-trained class conditional model, you can interpolate between the embeddings.

**Comment by takerum** [comment]
Hi, sorry for the late reply.
The interpolation code was fixed and is now available. The example is here:
```
python evaluations/gen_interpolated_images.py --n_zs=10 --n_intp=10 --snapshot=ResNetGeneratorImageNet_850000.npz --config=configs/sn_projection.yml --classes 986 989
```
Please let me know if anything is unclear.
